tracer/src/Datadog.Trace/Logging/Internal/DatadogRollingFileSink.cs

```diff
@@ -0,0 +1,249 @@
+// <copyright file="DatadogRollingFileSink.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
```

Comment:
`nullable enable` maybe? I'm guessing most of the Serilog code doesn't have that... also, if we're going to update to serilog 3.x soon, be aware we may need/want to update this soon